CHANGELOG.md

```diff
@@ -13,6 +13,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+
+
+## [1.18.1] - 2024-09-10
```

Comment:
Are we about to make a release? Otherwise this should be in the unreleased section..